src/sentry/nodestore/riak/client.py

```diff
@@ -187,14 +187,15 @@ class ConnectionManager(object):
 
     def __init__(
         self,
-        hosts=DEFAULT_NODES,
+        hosts=None,
         strategy=RoundRobinStrategy,
         randomize=True,
         timeout=3,
         cooldown=5,
         max_retries=None,
         tcp_keepalive=True
     ):
+        hosts = hosts or DEFAULT_NODES
```

Comment:
It kind of seems like this should only set the default if hosts is `None`? Otherwise an empty sequence will be overwritten with the default nodes instead of being caught with the assertion below, which seems kind of indirect to me at least.